### ISSUE TITLE
Unable to unbind hue dimmer with a zigate

### ISSUE THREAD

**Issue by Tonio16** [Koenkk/zigbee2mqtt#8857]
## What happened
I want to bind a hue bulb and a dimmer together. To do so, I have to unbind the dimmer from the coordinator, but it doesn’t work. Same issue for conbee describe in this topic:
https://github.com/Koenkk/zigbee2mqtt/issues/4171

## What did you expect to happen
That I can unbind coordinator from the remote.

## How to reproduce it (minimal and precise)
I expect that it’s due to the zigate.

## Debug info
Zigbee2MQTT version: 1.21.1
Adapter hardware: zigate v1
Adapter firmware 1d03

Here the log with herdsman one.
[log_debug.log](https://github.com/Koenkk/zigbee2mqtt/files/7219930/log_debug.log)

Antoine

**Comment by G1K** [comment]
Yes, I didn't think of that and pointed out the path to the source file. on end devices, the collected files are in dist and have the extension js

But in the final file, you need to search for the result by content.  The line number does not match. If you can't find it, then just wait for the update. Can you install the Dev version?
